test/specs/elements/Header/HeaderSubheader-test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import HeaderSubheader from 'src/elements/Header/HeaderSubheader'
+import * as common from 'test/specs/commonTests'
+
+describe('HeaderSubheader', () => {
+  common.rendersChildren(HeaderSubheader)
+  const wrapper = mount(<HeaderSubheader />)
+
+  it('renders a <div /> element', () => {
+    wrapper.should.have.tagName('div')
+  })
+
+  it('adds the sd-header-subheader class', () => {
+    wrapper.should.have.className('sd-header-subheader')
+  })
```

Comment:
My understanding was that HeaderSubheader isn't conforms to though as is a sub component and not exported on the top level stardust object?